changelogs/fragments/remove_jsonfile.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- moved jsonfile cache plugin back to ansible-base
```

Comment:
I don't think we need a changelog fragment for this. We didn't have a proper release yet (and currently the changelog describes changes since Ansible 2.9).

Reply:
Agreed, community.general hasn't been released yet.